static/app/components/organizations/projectSelector/selectorItem.tsx

```diff
@@ -71,98 +71,76 @@ function ProjectSelectorItem({
   };
 
   return (
-    <BadgeAndActionsWrapper>
-      <PageFilterRow
-        checked={isChecked}
-        onCheckClick={handleClick}
-        multi={multi}
-        renderCheckbox={({checkbox}) => (
-          <Feature
-            features={['organizations:global-views']}
-            hookName="feature-disabled:project-selector-checkbox"
-            renderDisabled={renderDisabledCheckbox}
-          >
-            {checkbox}
-          </Feature>
-        )}
-      >
-        <BadgeWrapper isMulti={multi}>
-          <IdBadge
-            project={project}
-            avatarSize={16}
-            displayName={<Highlight text={inputValue}>{project.slug}</Highlight>}
-            avatarProps={{consistentWidth: true}}
-            disableLink
-          />
-        </BadgeWrapper>
-        <StyledBookmarkStar
+    <ProjectFilterRow
+      checked={isChecked}
+      onCheckClick={handleClick}
+      multi={multi}
+      renderCheckbox={({checkbox}) => (
+        <Feature
+          features={['organizations:global-views']}
+          hookName="feature-disabled:project-selector-checkbox"
+          renderDisabled={renderDisabledCheckbox}
+        >
+          {checkbox}
+        </Feature>
+      )}
+    >
+      <BadgeWrapper>
+        <IdBadge
           project={project}
-          organization={organization}
-          onToggle={handleBookmarkToggle}
+          avatarSize={16}
+          displayName={<Highlight text={inputValue}>{project.slug}</Highlight>}
+          avatarProps={{consistentWidth: true}}
+          disableLink
         />
-        <StyledLink
-          to={`/organizations/${organization.slug}/projects/${project.slug}/?project=${project.id}`}
-          onClick={e => e.stopPropagation()}
-        >
-          <IconOpen />
-        </StyledLink>
-
-        <StyledLink
-          to={`/settings/${organization.slug}/${project.slug}/`}
-          onClick={e => e.stopPropagation()}
-        >
-          <IconSettings />
-        </StyledLink>
-      </PageFilterRow>
-    </BadgeAndActionsWrapper>
+      </BadgeWrapper>
+      <ActionBookmark
+        project={project}
+        organization={organization}
+        onToggle={handleBookmarkToggle}
+      />
+      <ActionButton
+        to={`/organizations/${organization.slug}/projects/${project.slug}/?project=${project.id}`}
+        size="zero"
+        priority="link"
+        aria-label="Project Details"
+        icon={<IconOpen />}
+      />
+      <ActionButton
+        to={`/settings/${organization.slug}/${project.slug}/`}
+        size="zero"
+        priority="link"
+        aria-label="Project Settings"
+        icon={<IconSettings />}
+      />
```

Comment:
Should we have a re-usable component that makes icon buttons? I bet we have some duplicate style definitions for this kind of component.

Reply:
IMO this is already close to being nice and succinct, but yeah we probably could make it even simpler

Reply:
Let's deal with it later?